index.html

```diff
@@ -173,126 +173,447 @@ <h2>
+                focus by the operating system's window manager:
+                <ol>
+                  1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                  Queue an element task</a> on the 
+                  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  user interaction task source</a>, given this, to perform the following steps:
+                  <ol>
+                    <li>
+                      <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockerror</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      <a>Reject</a> <a>promise</a> with a "{{WrongDocumentError}}" {{DOMException}}.
+                    </li>
+                    <li>
+                      Return <a>promise</a>.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                If the user agent's <a>pointer-lock target</a> is an element whose <a>shadow-including root</a> is
+                not equal to <a>this</a>'s <a>shadow-including root</a>, then:
+                <ol>
+                  1. <a href="https://html.spec.whatwg.org/multipage/webappapis.html#queue-an-element-task">
+                  Queue an element task</a> on the 
+                  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#user-interaction-task-source">
+                  user interaction task source</a>, given this, to perform the following steps:
+                  <ol type='i'>
+                    <li>
+                      <a href=https://dom.spec.whatwg.org/#concept-event-fire">Fire an event</a> named
+                      <a>pointerlockerror</a> at <a>this</a>'s 
+                      <a href="https://dom.spec.whatwg.org/#concept-node-document">node document</a>.
+                    </li>
+                    <li>
+                      <a>Reject</a> <a>promise</a> with a "{{InvalidStateError}}" {{DOMException}}.
+                    </li>
+                    <li>
+                      Return <a>promise</a>.
+                    </li>
+                  </ol>
+                </ol>
+              </li>
+              <li>
+                If the user agent's <a>pointer-lock target</a>'s <a>shadow-including
+                root</a> is equal to <a>this</a>'s <a>shadow-including root</a>, then:
+                <ol>
+                  <li>
+                    If <a>options</a> are equivalent to the current <a>pointer-lock options</a> then the <a>pointer-lock target</a>
+                    must be updated to this element, a <a>pointerlockchange</a>
+                    event sent, and the promise must <a>resolve</a>.
```

Comment:
This needs to use a similar style to the error cases:

1. If options are equivalent to the current pointer-lock options, then:
    1. Queue an element task on the user interaction task source, given this, to perform the following steps:
        1. Set the pointer-lock target to this.
        1. Fire an event named pointerlockchange at this's node document.
        1. Resolve promise.
    1. Return promise.